### ISSUE TITLE
Generated code contains Python 3 syntax error

### ISSUE THREAD

**Issue by takluyver** [enthought/comtypes#92]
comtypes generated code looking like this:

```python
# values for enumeration '_BoolValue'
True = 1
False = 0
BoolValue_Max = 2147483647
_BoolValue = c_int # enum
IHTMLDOMImplementation._methods_ = [
```

True and False are reserved words in Python 3, so this is a syntax error. It appears that the syntax error when loading the generated code made it repeatedly try to generate the code anew, slowing the application to a crawl. When I edited the cached file, commented out the two lines defining True and False, and re-ran, I stopped seeing errors and the code ran at a normal speed.

Scraps of information available on the web (e.g. [here](http://wutils.com/com-dll/constants/constants-MSHTML.htm)) indicate that `_BoolValue` is an enum provided by MSHTML. But I think that comtypes should do something to modify names that are Python keywords (e.g. `True_ = 1`? - I don't know what the constraints are).

It looks like the relevant bit of the code is `comtypes.tools.codegenerator.Generator.EnumValue`.

Here's the error I see:

```
# Generating comtypes.gen._3050F1C5_98B5_11CF_BB82_00AA00BDCE0B_0_4_0
Traceback (most recent call last):
  File "pkgs\comtypes\client\_events.py", line 117, in error_printer
    return func(*args, **kw)
  File "pkgs\webview\win32.py", line 253, in DocumentComplete
    custom_doc = self.browser.Document.QueryInterface(ICustomDoc)
  File "pkgs\comtypes\client\__init__.py", line 44, in wrap_outparam
    return GetBestInterface(punk)
  File "pkgs\comtypes\client\__init__.py", line 110, in GetBestInterface
    mod = GetModule(tlib)
  File "pkgs\comtypes\client\_generate.py", line 112, in GetModule
    mod = _CreateWrapper(tlib, pathname)
  File "pkgs\comtypes\client\_generate.py", line 188, in _CreateWrapper
    mod = _my_import(fullname)
  File "pkgs\comtypes\client\_generate.py", line 26, in _my_import
    return __import__(fullname, globals(), locals(), ['DUMMY'])
  File "C:\Users\Thomas\AppData\Roaming\Python\Python35\comtypes_cache\_3050F1C5
_98B5_11CF_BB82_00AA00BDCE0B_0_4_0.py", line 23290
    True = 1
           ^
SyntaxError: can't assign to keyword
ERROR:comtypes._comobject:Exception in DWebBrowserEvents2.DocumentComplete imple
mentation:
Traceback (most recent call last):
  File "pkgs\comtypes\_comobject.py", line 147, in call_without_this
    result = mth(*inargs)
  File "pkgs\comtypes\client\_events.py", line 117, in error_printer
    return func(*args, **kw)
  File "pkgs\webview\win32.py", line 253, in DocumentComplete
    custom_doc = self.browser.Document.QueryInterface(ICustomDoc)
  File "pkgs\comtypes\client\__init__.py", line 44, in wrap_outparam
    return GetBestInterface(punk)
  File "pkgs\comtypes\client\__init__.py", line 110, in GetBestInterface
    mod = GetModule(tlib)
  File "pkgs\comtypes\client\_generate.py", line 112, in GetModule
    mod = _CreateWrapper(tlib, pathname)
  File "pkgs\comtypes\client\_generate.py", line 188, in _CreateWrapper
    mod = _my_import(fullname)
  File "pkgs\comtypes\client\_generate.py", line 26, in _my_import
    return __import__(fullname, globals(), locals(), ['DUMMY'])
  File "C:\Users\Thomas\AppData\Roaming\Python\Python35\comtypes_cache\_3050F1C5
_98B5_11CF_BB82_00AA00BDCE0B_0_4_0.py", line 23290
    True = 1
           ^
SyntaxError: can't assign to keyword
```

**Comment by yoavram** [comment]
:+1: 
Same here on Win7 64-bit with Python 3.4.4 and comtypes 1.1.2. It doesn't happen with Py 2.7.

**Comment by laripk** [comment]
Here is my attempt to fix this: #95

**Comment by takluyver** [comment]
#95 was merged, so I'm closing this - I haven't tested, so I'm assuming the fix is good. Any chance of a new release with the fix?
